Makefile

```diff
@@ -129,10 +130,10 @@ $(SWAGGER): $(TOOLS_DIR)/go.mod
 # help
 ##################
 
-help: # Display help
+help: ## Display help
```

Comment:
This was broken. I copied the new working version of this from Fulcio's Makefile [here](https://github.com/sigstore/fulcio/blob/main/Makefile#LL152-L156C28).